traefik/templates/_podtemplate.tpl

```diff
@@ -20,6 +20,9 @@
       serviceAccountName: {{ include "traefik.serviceAccountName" . }}
       terminationGracePeriodSeconds: 60
       hostNetwork: {{ .Values.hostNetwork }}
+      {{- if .Values.topologySpreadConstraints }}
```

Comment:
Shouldnt that be behind a version check? What will happen on installations prior to 1.19?

Reply:
I would like to add a check `(semverCompare ">=1.19.0" .Capabilities.KubeVersion.Version)` but helm unittest plugin does not support setting .Capabilities and a lot of tests fail. My idea was that someone, who wants to use this feature should now whether or not it is enabled on his/her cluster

Reply:
Do you know what happens when someone "accidentally" activates that? Like what is the potential error?

Reply:
It will fail during installation/upgrade

Reply:
Then, I think, we should rather hide it behind a version compare and don't have a test for it. >/ As sad as it is, I think better user experience is to favor over better  test coverage on that part.